storage/cloud-client/storage_unspecify_public_access_prevention.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+# [START storage_unspecify_public_access_prevention]
+from google.cloud import storage
+from google.cloud.storage.bucket import PUBLIC_ACCESS_PREVENTION_UNSPECIFIED
+
+
+def unspecify_public_access_prevention(bucket_name):
+    """Unspecify public access prevention for a bucket"""
```

Comment:
I think it's not that useful to have a docstring that just restates the function name basically. More relevant here is that this removes public access prevention for the bucket (so we will instead inherit the org-level policy). I guess we should work with Karen to clarify what should go in the docstring and what should go in the text above the sample block, though.

Reply:
Good point, I will follow up with Karen on this.

Reply:
+1, something along the lines of "Sets the public access prevention status to unspecified, so that the bucket inherits its setting from its parent project."